src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/ReducibleAnnotation.java

```diff
@@ -0,0 +1,27 @@
+package org.broadinstitute.hellbender.tools.walkers.annotator;
+
+import htsjdk.variant.variantcontext.VariantContext;
+import org.broadinstitute.hellbender.engine.ReferenceContext;
+import org.broadinstitute.hellbender.utils.genotyper.PerReadAlleleLikelihoodMap;
+
+import java.util.Map;
+
+/**
+ * An interface for annotations that are calculated using raw data across samples, rather than the median (or median of median) of samples values
+ */
+public interface ReducibleAnnotation extends Annotation {
```

Comment:
Add comment that `ReducibleAnnotation` currently lacks any `reduce` functionality, since it hasn't been ported (interface is bizarre/confusing without this comment).